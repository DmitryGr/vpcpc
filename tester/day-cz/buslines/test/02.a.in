100
29 96
62 25
32 80
33 91
67 42
71 1
33 39
19 68
47 77
29 75
93 78
18 45
56 91
50 92
40 54
36 22
91 8
87 12
14 75
81 68
40 66
5 14
43 57
32 42
64 78
57 97
59 9
61 56
33 44
95 90
16 83
74 60
76 87
58 12
61 20
49 46
78 77
25 16
15 3
77 36
12 43
51 2
96 6
89 53
9 77
29 21
84 61
63 88
42 23
73 40
10 75
56 45
20 99
55 40
43 56
79 33
39 89
55 27
13 41
42 78
4 89
44 35
39 74
95 71
3 65
39 11
18 98
42 55
21 85
30 14
53 72
23 17
21 63
2 46
67 69
19 40
50 82
86 8
94 60
74 42
34 85
74 21
21 41
38 25
91 71
37 67
24 68
25 8
56 65
100 9
91 70
29 31
27 49
52 42
45 28
7 96
26 18
50 4
22 48
