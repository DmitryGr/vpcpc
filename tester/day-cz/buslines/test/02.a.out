99
197
197
387
99
99
99
662
294
99
99
389
99
294
99
197
99
294
387
197
1499
197
197
99
484
99
479
99
932
99
99
197
2468
99
99
293
99
99
2832
758
197
2654
665
197
482
293
99
99
387
294
99
99
197
99
1257
1750
197
99
99
197
389
99
197
99
293
99
294
294
99
99
389
99
99
3217
482
99
758
1006
99
99
99
99
99
99
197
99
197
99
665
99
2522
99
99
99
197
294
99
99
99
99
